ml-agents-protobuf/make.bat

```diff
@@ -0,0 +1,43 @@
+# variables
+
+# GRPC-TOOLS required. Install with `nuget install Grpc.Tools`. 
+# Then un-comment and replace [DIRECTORY] with location of files.
+# For example, on macOS, you might have something like:
+COMPILER=Grpc.Tools.1.14.1/tools/macosx_x64
+# COMPILER=[DIRECTORY]
+
+SRC_DIR=proto/mlagents/envs/communicator_objects
+DST_DIR_C=cs/CommunicatorObjects
```

Comment:
Is there a way to generate directly into the appropriate folders ?

Reply:
Yep, now it automatically puts the proto files into the right folders.